### PR TITLE
Fixed #29646 -- Doc'd the validators that each model and form field uses.

### PR DESCRIPTION
**Pull request by jeffyancey**
https://code.djangoproject.com/ticket/29646

### CODE REVIEW

**Comment by jeffyancey** [comment]
@Miguelacosta90 looks like there is a link to unsubscribe at the bottom of the email you pasted.

**Comment by Miguelacosta90** [comment]
@jeffyancey thank you

**Comment by jeffyancey** [comment]
i see we use sphinx-build to somehow test our docs. If anyone knows off the top of their head a convenient way for me to run a similar test locally in the future, I'd sure appreciate it.

**Comment by timgraham** [comment]
[Writing documentation](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/) should answer your question.

Don't worry about non-deterministic failures on non-docs builds for this change.

You've addressed the issue for form fields but not for model fields.

**Comment by jeffyancey** [comment]
Thanks for the response, I'll go find what fields I missed in the models docs

File: docs/ref/models/fields.txt [hunk 1]
```diff
@@ -270,7 +269,7 @@ desire. For example::
     help_text="Please use the following format: <em>YYYY-MM-DD</em>."
 
 Alternatively you can use plain text and
-:func:`django.utils.html.escape` to escape any HTML special characters. Ensure
+``django.utils.html.escape()`` to escape any HTML special characters. Ensure
```
Comment (carltongibson) [inline docs/ref/models/fields.txt]: Why remove the `:func:` cross-reference?

**Pull request merged**